### PR TITLE
fix: keep local signal options from interfering with global options

### PR DESCRIPTION
**Pull request by maxfrigge**
Discussed in #239

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -86,21 +86,26 @@ var Controller = function (Model, services) {
...
+        var signalOptions = Object.keys(signals[name]).reduce(function (signalOptions, key) {
+          if (key !== 'chain') {
+            signalOptions[key] = signals[name][key]
           }
-          return options
-        }, options || {})
-        signal(key, signals[key].chain, options)
+          return signalOptions
+        }, optionsCopy)
+        console.log(options, optionsCopy, signalOptions, signals[name].chain)
```
Comment (Guria) [inline src/index.js]: remove it

**Comment by Guria** [comment]
@maxfrigge would you like to secure it with test?

File: src/index.js (hunk 1 above)
Comment (maxfrigge) [inline src/index.js]: crap :) missed that one. thanks!

**Comment by Guria** [comment]
LGTM
cc @christianalfoni

**Comment by christianalfoni** [comment]
Looks great! :-) Thanks @maxfrigge !

**Pull request merged**